airbyte-integrations/connectors/source-open-exchange-rates/integration_tests/sample_state.json

```
@@ -0,0 +1,5 @@
+{
+  "exchange_rates": {
+    "date": "2022-11-13"
```

We'll have to put a timestamp here as well

```suggestion
    "timestamp": 1673379575
```